src/app/person-profile/person-profile.component.html

```diff
@@ -1,42 +1,35 @@
 <table mat-table [dataSource]="dataSource" class="mat-elevation-z8">
 
-  <!-- Name Column -->
```

Comment:
Goodbye to all the useless Name Column placeholders from copy and pasting!

Reply:
yay